src/Workspaces/Core/Portable/SymbolKey/SymbolKey.FunctionPointerTypeSymbolKey.cs

```diff
@@ -16,7 +18,7 @@ public static void Create(IFunctionPointerTypeSymbol symbol, SymbolKeyWriter vis
                 visitor.WriteParameterTypesArray(symbol.Signature.Parameters);
             }
 
-            public static SymbolKeyResolution Resolve(SymbolKeyReader reader, out string failureReason)
+            public static SymbolKeyResolution Resolve(SymbolKeyReader reader, out string? failureReason)
```

Comment:
_Maybe_ my feature branch won't conflict? I don't have hope though.

Reply:
@333fred Good call, we'll see how we want to sequence our two changes.

Reply:
I don't think it matters too much, it'll be a minor conflict resolution.